Cargo.toml

```diff
@@ -16,10 +16,12 @@ members = [
   "rusqlite_migration_tests",
   "examples/*",
 ]
-
 # https://doc.rust-lang.org/cargo/reference/resolver.html#feature-resolver-version-2
 resolver = "2"
 
+[patch.crates-io]
+tokio-rusqlite = { git = "https://github.com/programatik29/tokio-rusqlite.git" }
```

Comment:
I understand that we can’t publish a new version to crates.io with this, so this patch will need to go before we merge this PR. But it helps to make progress and test things.

Reply:
Updated tokio-rusqlite, this is now unnecessary.